src/actions/user.actions.js

```diff
@@ -0,0 +1,32 @@
+import { userConstants } from "../constants/user.constants";
+import { userService } from "../services/user.services";
+import { history } from "../helpers/history";
+import { alertActions } from "../actions/alert.actions";
+
+export const userActions = {
+    login,
+};
+
+// Actions creator, the alerts will be removed
+function login(email, password) {
+    return dispatch => {
+        dispatch(request({ email }));
+
+        userService.login(email, password)
```

Comment:
This is fine. Take a look at _async - await_ syntax later, it will make these actions far more readable. https://javascript.info/async-await